packages/prettier-plugin-java/src/printers/blocks-and-statements.js

```diff
@@ -49,18 +49,23 @@ class BlocksAndStatementPrettierVisitor {
 
   localVariableDeclarationStatement(ctx) {
     const localVariableDeclaration = this.visit(ctx.localVariableDeclaration);
-    return rejectAndConcat([localVariableDeclaration, ctx.Semicolon[0]]);
+    return group(rejectAndConcat([localVariableDeclaration, ctx.Semicolon[0]]));
```

Comment:
I don't think there is the need for a group here, as there are no line between the localVariableDeclaration and the semicolon.

Reply:
Fixed, thank you it seems I forgot to remove it.